m3/src/main/java/com/uber/m3/tally/m3/M3Reporter.java

```diff
@@ -99,20 +99,23 @@
 
     private static final int MIN_METRIC_BUCKET_ID_TAG_LENGTH = 4;
 
+    /**
```

Comment:
can we just remove any code assuming NUM_PROCESSORS >1 at this point?

i.e. remove this var and change `List<Processor> processors` to a `Processor processor`?

Reply:
Please feel free to pick it up

Reply:
sg